src/ipc/ipc4/handler.c

```diff
@@ -1607,6 +1577,7 @@ void ipc_cmd(struct ipc_cmd_hdr *_hdr)
 	/* no process on scheduled thread */
 	atomic_set(&msg_data.delayed_reply, 0);
 	msg_data.delayed_error = 0;
+	msg_reply.tx_data = NULL;
```

Comment:
not particularly against this change, but just for clarity - this is just "cleaning up," right? Without it it would work just as well?

Reply:
Yes, this is just cleaning, without this change everything should work. The code only checks the `tx_size` field.